core/domain/exp_domain.py

```diff
@@ -4891,7 +4894,7 @@ def fix_content(cls, html: str) -> str:
         html = cls._fix_rte_tags(
             html, is_tags_nested_inside_tabs_or_collapsible=False)
         html = cls._fix_tabs_and_collapsible_tags(html)
-        return html
+        return html.replace('\xa0', '&nbsp;')
```

Comment:
@kevintab95, Please note that I have also updated the part where `&nbsp;` was converting to `\xa0`. Thanks

Reply:
(Just curious) It will replace an actual handwritten text "\xa0" with &nbsp. (Not sure what's the RTE's output for user writing \xa0 in the paragraph.)

Reply:
I'm not completely sure on what would happen but I can try it out and update you. I will try this out locally and will update you.
Thanks

Reply:
The RTE output will be something like `\\xa0` which I think will be counted as a simple string instead of a special value.
Thanks